### DISCUSSION TITLE
Have auto-merge always enabled

### DISCUSSION THREAD

**Discussion by Pandapip1** [community/community#28365] [Pull Requests]
I am aware that it is possible to do this with an action, but it would be really helpful if there was an option (maybe in the branch protection settings?) to have auto-merge **always** enabled.

**Comment by jfullstackdev** [comment]
maybe this one 

https://github.com/marketplace/actions/merge-pull-requests-automerge-action

**Reply by Pandapip1** [reply]
It works, but it pollutes the comment history like crazy.
